clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -385,27 +385,19 @@ protected void handleUserCommand() {
       boolean force = line.hasOption(GravitinoOptions.FORCE);
       newDeleteUser(url, ignore, force, metalake, user).handle();
     } else if (CommandActions.REVOKE.equals(command)) {
+      // Retrieve values of the role option from the command line, e.g., ["role1", "role2"]
```

Comment:
There is no need for comments like this that say what the code is doing. In general comments should explain why design choices were made or explain complex code. Often comments like this tend to get out of sync.

Reply:
Should comments be added just once or every time the role option is accessed？

Reply:
I would remove the comment it doesn't add much as the code is quite simple and easy to understand

Reply:
Hi Justin, I change the comments, plz  review this PR again when you have time.